defaults/main.yml

```diff
@@ -1 +1,49 @@
 # defaults file for nfs_server
+
+# List of NFS shares to create in exports file
+nfs_server_shares: []
+# Example:
+# nfs_server_shares:
+#     # path to share (required)
+#   - share_path: /srv/share1
+#     # IP networks for which to allow access (required)
+#     host_allow: '*'
+#     # Comma-separated list of mount options (required)
+#     opts: rw,sync,all_squash
+#     # create the share directory (optional)
+#     create_dir: true
+#     # owner of the NFS share directory (optional)
+#     owner: nfsnobody
+#     # group of the NFS share directory (optional)
+#     group: nfsnobody
+#     # mode of the NFS share directory (optional)
+#     # NOTE: Even though this is an integer value, it still must be quoted
+#     # here due to the linked issue below:
+#     # https://github.com/ansible/ansible/issues/31952
+#     mode: '0755'
+#   - share_path: /srv/share2
+#     host_allow: '192.168.0.0/24'
+#     opts: ro,sync
+
+# Packages to install for the NFS server
+nfs_server_packages:
+  - nfs-utils
+
+# Services to enable and start for the NFS server
+nfs_server_services:
+  # required for NFSv3
+  - nfs
+  # required for NFSv3, not used by NFSv4
```

Comment:
Indeed it would be, and ideally we could have something like a `nfs_version` variable that would set all this on behalf of the user.  The docs for NFS are really murky, and its employment is not at all intuitive.  I think we could create a future improvement ticket to dig into all that.  At least for now, the user can decide which NFS version components they want to specify.  Thoughts?

Reply:
I agree on an issue to tackle this later. Ideally, we would query the rpmdb to find out what version was installed and the role would just know what to do in the appropriate environment.

Reply:
Ticket created:

https://github.com/oasis-roles/nfs_server/issues/2